### ISSUE TITLE
Very low PSNR (<10) on test set despite validation PSNR >23 during training

### ISSUE THREAD

**Issue by l298555-ee** [sanechips-multimedia/syenet#12]
Hi authors, thank you for open-sourcing SYENet.
I am reproducing your results on the MAI 2022 dataset using exactly the original repository without any code changes.
Command I use for training:
python main.py -task train -model_task isp -device cuda

I train for 350 epochs as mentioned in the MAI challenge report. During training, the validation PSNR consistently goes above 23 dB (which looks reasonable).
However, when I evaluate on the official MAI test set (using python main.py -task test -model_task isp -device cuda), the PSNR drops drastically to below 10 dB (sometimes even ~7–8 dB). This huge gap between validation and test performance does not make any sense.

Things I have double-checked:

- Using the official MAI dataset (train+validation). As test dataset was not provided, I have divided training dataset into 10% test and 90% train dataset.
- No data augmentation or preprocessing changes
- Same random seed behavior (default)
- No modifications in the model architecture or loss functions

Could you please let me know what might be causing this extreme drop on the test set? Is there any special testing script, cropping/padding rule, or post-processing step that must be applied specifically for the MAI test set? Any advice would be greatly appreciated.

Thank you!
